src/Middleware/OutputCaching/src/Policies/DefaultPolicy.cs

```diff
@@ -46,15 +46,13 @@ ValueTask IOutputCachePolicy.ServeResponseAsync(OutputCacheContext context, Canc
         // Verify existence of cookie headers
         if (!StringValues.IsNullOrEmpty(response.Headers.SetCookie))
         {
-            context.Logger.ResponseWithSetCookieNotCacheable();
```

Comment:
Removed logs from this policy as it requires the logger to be resolved (on each request?) and also because further policies can negate the results here and the lot would be wrong.